packages/@lwc/template-compiler/src/shared/types.ts

```diff
@@ -223,3 +217,7 @@ export enum ElementDirectiveName {
     Spread = 'lwc:spread',
     Key = 'key',
 }
+export enum RootDirectiveName {
+    PreserveComments = 'lwc:presere-comments',
```

Comment:
```suggestion
    PreserveComments = 'lwc:preserve-comments',
```

It's worrisome that this typo wasn't caught by any of the tests. Maybe we should get rid of this `const` and replace it with the `enum`:

https://github.com/salesforce/lwc/blob/60ea25a0a3e23fa9da0ab4eda70147b2008ce9ae/packages/%40lwc/template-compiler/src/parser/constants.ts#L34-L37

Reply:
I can fix this in my PR, no worries.

Reply:
Done: [d937949](https://github.com/salesforce/lwc/pull/3067/commits/d9379491baf887982ee5c3fffc49c9b5edb6e972)